tools/langs/cplusplus_vrv.py

```diff
@@ -882,32 +917,27 @@ def __create_att_classes(schema, outdir, includes_dir):
     fmi.close()
     fmi_cpp.write(CONVERTER_IMPL_TEMPLATE_END)
     fmi_cpp.close()
-    lg.debug("\tCreated attconverter.h/cpp".format(module.lower()))
+    lg.debug("\tCreated attconverter.h/cpp")
 
 def parse_includes(file_dir, includes_dir):
     lg.debug("Parsing includes")
-
     # get the files in the includes directory
-    includes = [f for f in os.listdir(includes_dir) if not f.startswith(".") and f.endswith(".inc")]
-    # currently unused, see below comment in __copy_codefile
-    # copies = [f for f in os.listdir(includes_dir) if f.endswith(".copy")]
+    includes = [f for f in Path(includes_dir).iterdir()
+                if not f.name.startswith(".")]
 
-    for dp,dn,fn in os.walk(file_dir):
-        for f in fn:
-            if f.startswith("."):
-                continue
-            methods, inc = __process_include(f, includes, includes_dir)
-            if methods:
-                __parse_codefile(methods, inc, dp, f)
+    for f in Path(file_dir).iterdir():
+        if f.name.startswith("."):
+            continue
+        methods, inc = __process_include(f, includes, includes_dir)
+        if methods:
+            __parse_codefile(methods, inc, f.parent, f)
 
 def __process_include(fname, includes, includes_dir):
-    name,ext = os.path.splitext(fname)
     new_methods, includes_block = None, None
     if "{0}.inc".format(fname) in includes:
         lg.debug("\tProcessing include for {0}".format(fname))
-        f = open(os.path.join(includes_dir, "{0}.inc".format(fname)), 'r')
-        includefile = f.read()
-        f.close()
+        f = Path(includes_dir, "{0}.inc".format(fname))
```

Comment:
I wouldn't call a path variable `f` as it strongly suggests to contain something like a file. If you never use it again, just make it a one-liner:
```python
includefile = Path(includes_dir, "{0}.inc".format(fname)).read_text()
```

Reply:
Thanks! Done.